src/main/java/com/parasoft/findings/utils/rules/RuleParserUtil.java

```diff
@@ -59,7 +59,7 @@ public static void saxParse(InputStream in, DefaultHandler handler, RuleParsingO
             return;
         }
 
-        XMLReader reader = createReader(options);
+        XMLReader reader = createReader(options); // parasoft-suppress OWASP2021.A5.DXXE "The XML reader is correctly configured in createReader() method."
```

Comment:
It seems that both `disableDTD` and `disableExternalEntities` need to be set to true to resolve the violation. You may want to modify the logic to always set these two options to true.

Reply:
The changes we made suppose to fix the issue and the suppression should be deleted in this case.
Please also simplify the code logic since we'll use fixed value as `true` for `disableDTD` and `disableExternalEntities`